src/js/vocabulary.js

```diff
@@ -1,32 +1,35 @@
-const exploreButton = document.querySelector('button.explore');
-const explorePanel = document.querySelector('.explore-panel');
+const exploreButton = document.querySelector("button.explore");
```

Comment:
this PR contains style adjustments where you changes from `'` to `"`, please revert these changes. We try to keep stylization changes in their own Issues and PRs.